CONTRIBUTING.md

```
@@ -0,0 +1,289 @@
+go get <module>@<version>
+go mod tidy
+make verify
+
+# Build container image (requires podman or imagebuilder)
+IMAGE_BUILD_BUILDER="podman build" IMAGE_BUILD_DEFAULT_FLAGS="" make image-origin-oauth-server
+
+# Run locally (requires kubeconfig with OAuth resource permissions)
+./oauth-server osinserver --config=/path/to/config.yaml --v=4
+```
+
+**Note**: Container image builds require either `podman` or `imagebuilder` installed. See [Build troubleshooting](#build-troubleshooting) if you encounter issues.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)